resources/content/db/changelog.xml

```diff
@@ -60,4 +60,5 @@
     <include file="db/core-053.xml"/>
     <include file="db/core-054.xml"/>
     <include file="db/core-055.xml"/>
+    <include file="db/vip.xml"/>
```

Comment:
@ibuildthecloud :) I always rename after the final rebase. The day I miss it, I'll owe you $10